strings/title.py

```diff
@@ -0,0 +1,60 @@
+def convert_word(word: str) -> str:

```

Comment:
Let's have a self-documenting function name.
```suggestion
def to_title_case(word: str) -> str:
```

Reply:
@cclauss Hey, Actually I am using two methods:
convert_word will convert a single word to a title case word ex: "dog" to "Dog"
THEN main method to_title_case will take all the converted words and will join them together to create a sentence.

so just wanted to let you know we already have a method name to_title_case so maybe I can give some other name to convert_word method like to_title_word or something else

Reply:
Let's keep the word one short: `to_title_case()` and the sentence one longer: `sentence_to_title_case()` and we can leave the word `convert` out of the function names.